content/doc/addons/cellar.md

```diff
@@ -456,7 +456,7 @@ Here this configuration has two `CORS` rules:
 
 {{< callout type="info" >}}
 
-**Updating the CORS configuration replaces the old one**  
+**Updating the CORS configuration replaces the old one**
 If you update your CORS configuration, the new configuration replaces the old one. Be sure to save it before you update it if you ever need to rollback.
```

Comment:
```suggestion
**Updating the CORS configuration replaces the old one**  
If you update your CORS configuration, the new configuration replaces the old one. Be sure to save it before you update it if you ever need to rollback.
```
I find that the two space are necessary. It make the line in Bold be on top of the other one.
Otherwise, if you look at the review app, you have something that look like this :  

**Updating the CORS configuration replaces the old one** If you update your CORS configuration, the new configuration replaces the old one. Be sure to save it before you update it if you ever need to rollback.

I think this look weird, because you have a new sentences that start in the middle of the line. Having them on top of each other look and read better.

Reply:
Yep, to explain what's going on here: my IDE trims automatically unnecessary space (those at the end of a line for example), it's why this change was done. 

But as it's Markdown, a single new line is not a new paragraph. So both texts were already next to each other once rendered in HTML. Without these spaces they're glued together, with these spaces, there are two spaces between them. 

The solution here is to add a space but not a new line (as it's not rendered). There's usually no “title” in a callout so I added an extra `:` to make it more logic. 

As it's a minor change I didn't do it in a separate commit, but I can if you prefer.